tools/deployment/package/docker-compose.base.yaml

```diff
@@ -0,0 +1,369 @@
+name: "clp-package-base"
+
+# Common service defaults.
+x-service-defaults: &service_defaults
+  image: "${CLP_PACKAGE_CONTAINER_IMAGE_REF:-clp-package}"
+  logging:
+    driver: "local"
+  stop_grace_period: "3s"
+  user: "${CLP_FIRST_PARTY_SERVICE_UID_GID:-1000:1000}"
+
+# Common healthcheck defaults.
+x-healthcheck-defaults: &healthcheck_defaults
+  # Avoid lowering to prevent excessive resource usage.
+  interval: "30s"
+
+  # Mark unhealthy after 3 failed probes.
+  # - In steady state, (<interval> + <timeout>) × 3 = ~90s before the service is marked unhealthy.
+  # - From startup, <start_period> (60s) + ~90s = ~150s before the service is marked unhealthy.
+  retries: 3
+
+  # Frequent checks during startup allow fast transition to healthy.
+  start_interval: "2s"
+
+  # Ignore failures for <start_period> / (<start_interval> + <timeout>) = ~15 frequent checks before
+  # counting retries.
+  start_period: "60s"
+
+  # Short timeout since no remote communication is expected.
+  timeout: "2s"
+
+# Common volume definitions.
+x-volume-definitions:
+  aws-config-readonly: &volume_aws_config_readonly
+    type: "bind"
+    source: "${CLP_AWS_CONFIG_DIR_HOST:-~/.aws}"
+    target: "/.aws"
+    read_only: true
+  clp-config-readonly: &volume_clp_config_readonly
+    type: "bind"
+    source: "${CLP_LOGS_DIR_HOST:-./var/log}/.clp-config.yml"
+    target: "/etc/clp-config.yml"
+    read_only: true
+  clp-logs: &volume_clp_logs
+    type: "bind"
+    source: "${CLP_LOGS_DIR_HOST:-./var/log}"
+    target: "/var/log"
+  logs-input-readonly: &volume_logs_input_readonly
```

Comment:
Shall we make this conditional based on whether `CLP_LOGS_INPUT_DIR_HOST` is set? Otherwise, we'll be mounting the user's entire filesystem even though they're using S3 as input.